docs/Installation/Installation.md

```diff
@@ -58,7 +58,9 @@ all of these dependencies.
 #### Required:
 
 * [GCC](https://gcc.gnu.org/) 9.1 or later,
-[Clang](https://clang.llvm.org/) 11.0 or later, or AppleClang 11.0.0 or later
+[Clang](https://clang.llvm.org/) 12.0 or later (see
+[here](https://apt.llvm.org/) for how to get newer versions of clang through
+apt), or AppleClang 11.0.0 or later
```

Comment:
You can bump AppleClang to 13 as well, since that's the version we're testing on CI.